modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-creating-ostree-iso_{context}"]
+= Creating the {op-system-ostree-first} image
+
+Use the following procedure to create the ISO. The {op-system-ostree} Installer image pulls the commit from the running container and creates an installable boot ISO with a Kickstart file configured to use the embedded OSTree commit.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
+* You have the `podman` tool.
+
+.Procedure
+
+. Start an `ostree` container image build by running the following command:
++
+[source,terminal]
+----
+BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/9/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
+----
```

Comment:
In the rendered output, this step is coming before the step that creates the `minimal-microshift.toml` file and pushes it up to the composer API.

Reply:
@dhellmann I am not following. Are steps out of order?

Reply:
Yes. This step says to use a blueprint ("minimal-microshift") that has not been created in the workflow up to that point.

Reply:
I think the order is correct because `minimal-microshift` has already been made available to composer-cli here:
https://github.com/openshift/openshift-docs/pull/61003/files#diff-be6a1b04f4fe3b1a11c5f4e02cfc56d258efcb6fac9405cc51cd001e26b82a07R50